client/src/partials/journal/journal.html

```diff
@@ -17,11 +17,13 @@
         <button class="btn btn-default" data-method="configure" ng-click="JournalCtrl.openColumnConfigModal()">
           <i class="fa fa-cogs"></i> {{ "FORM.BUTTONS.CONFIGURE" | translate }}
         </button>
+
+        <button class="btn btn-default" data-method="trial" ng-disabled="JournalCtrl.grouping.selectedRowCount <= 0" ng-click="JournalCtrl.openTrialBalanceModal()">
```

Comment:
Lol, the button of rapprochement will be active if there is at leat one row selected so if it is ```>0 ``` or ```>=1``` that is okay.
I can change it to ```ng-disabled="JournalCtrl.grouping.selectedRowCount <1"``` but it is almost the same than ```ng-disabled="JournalCtrl.grouping.selectedRowCount <=0"```. I am changing that.